docker/Dockerfile

```diff
@@ -14,17 +14,45 @@ RUN apt-get update \
     # Install google-fluentd at version 1.5.35-1.
     && curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_SUFFIX=20180810-1 DO_NOT_INSTALL_CATCH_ALL_CONFIG=true bash \
     # Clean up to reduce image size.
-    && apt-get remove -y \
+    && apt-get purge -y \
         curl \
         lsb-release \
         build-essential \
+        gtk2.0 \
+        adduser \
+        libkrb5-3 \
     && rm -rf /var/lib/apt/lists/* /tmp/* \
-    && apt-get autoremove -y \
+    && apt-get autoremove -y --purge \
     && find /var/log -name "*.log" -type f -delete \
+    # Remove .a .c .cc .h files and include libraries
+    && rm -rf /opt/google-fluentd/embedded/include /opt/google-fluentd/embedded/lib/*.a \
+    && find /opt/google-fluentd/embedded/ -name '*.c'  | xargs rm \
+    && find /opt/google-fluentd/embedded/ -name '*.cc' | xargs rm \
+    && find /opt/google-fluentd/embedded/ -name '*.h'  | xargs rm \
     # Remove docs.
-    && rm -rf /usr/share/doc /usr/share/man /opt/google-fluentd/embedded/share/doc /opt/google-fluentd/embedded/lib/ruby/gems/*/doc ./opt/google-fluentd/embedded/lib/ruby/*/rdoc /opt/google-fluentd/embedded/share/gtk-doc /opt/google-fluentd/embedded/share/man /opt/google-fluentd/embedded/share/terminfo \
-    # Remove gems that are only needed when ingesting logs to Treasure Data Cloud.
-    && /opt/google-fluentd/embedded/bin/gem uninstall td td-client td-logger fluent-plugin-td fluent-plugin-td-monitoring hirb parallel ohai mixlib-cli mixlib-config mixlib-log mixlib-shellout systemu -ax --force \
+    && rm -rf \
+        /usr/share/doc \
+        /usr/share/man \
+        /opt/google-fluentd/embedded/share/doc \
```

Comment:
Why not remove the `rdoc` files as well?

Reply:
`rdoc` is a default gem. Without it, `gem` won't install any gems.

Reply:
Sorry, I meant the `rdoc` files (i.e., the gem docs), not the `rdoc` gem.

Reply:
Updated.